plugin/pkg/utils/miscellaneous.go

```diff
@@ -24,7 +24,8 @@ import (
 	"k8s.io/apiserver/pkg/admission"
 
 	"github.com/gardener/gardener/pkg/apis/core"
-	gardencorelisters "github.com/gardener/gardener/pkg/client/core/listers/core/internalversion"
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	gardencorelisters "github.com/gardener/gardener/pkg/client/core/listers/core/v1beta1"
```

Comment:
Should this not be called `gardencorev1beta1listers`?
Similarly in other places.

Reply:
As we no longer have the internal listers and informers. I chose to name it it `gardencorelisters`. WDYT?

Reply:
I think for consistency reasons, we should use v1beta1 in the name. What if in the future there is another version?